docs/readme.md

```diff
@@ -0,0 +1,83 @@
+# Widgets
+
+## base nodes
+* [Node](/docs/widgets/baseNodes/node.md) (abstract)
```

Comment:
> t looks like github actually resolves these urls correctly

Interesting... probably it resolves to the root of the repo...


> A few of the links are camel case where the file is all lower and that breaks it but it is navigable as is

I think they should be checked and fixed :-/